role-merge-prs.sh

```diff
@@ -39,33 +39,63 @@ list_prs() {
     gh pr list -R "$origin_org/$repo" "${searchargs[@]}" "$@"
 }
 
-get_check_summary() {
-    local pr total success failure pending cancelled error conclusion state name
+get_checks() {
+    local pr total success failure pending cancelled error conclusion state name status display_name
     pr="$1"
     gh pr view "$pr" -R "$origin_org/$repo" --json statusCheckRollup \
-      --template '{{range .statusCheckRollup}}{{print (or .conclusion "<nil>")}} {{print (or .state "<nil>")}} {{println .name}}{{end}}' | {
-    while read -r conclusion state name; do
+      --template '{{range .statusCheckRollup}}{{print (or .conclusion "<nil>")}}#{{print (or .state "<nil>")}}#{{print .name}}#{{println .context}}{{end}}' | \
+    while IFS="#" read -r conclusion state name context; do
```

Comment:
Does this assume that PRs do not have `#` in their name?

Reply:
not PRs, but the labels/text we use for the checks/tests - e.g. 
```json
    {
      "__typename": "CheckRun",
      "completedAt": "2023-05-23T00:44:11Z",
      "conclusion": "SUCCESS",
      "detailsUrl": "https://github.com/linux-system-roles/network/actions/runs/5051822786/jobs/9064065108",
      "name": "ansible_lint",
      "startedAt": "2023-05-23T00:42:44Z",
      "status": "COMPLETED",
      "workflowName": "Ansible Lint"
    }
```
or
```json
    {
      "__typename": "StatusContext",
      "context": "CentOS-7-latest/ansible-2.14",
      "startedAt": "2023-05-23T22:46:19Z",
      "state": "FAILURE",
      "targetUrl": "https://dl.fedoraproject.org/pub/alt/linuxsystemroles/logs/lsr-citool_network-611-9c5b88b_CentOS-7-latest_20230523-224601/artifacts/summary.html"
    }
```
right now it seems pretty safe to use `#` as the delimiter - doesn't look like any `name` or `context` uses that character